src/LibPQ.jl

```diff
@@ -718,24 +844,52 @@ function column_name(jl_result::Result, column_number::Integer)
 end
 
 """
-    column_names(jl_result::Result, column_number::Integer) -> Vector{String}
+    column_names(jl_result::Result) -> Vector{String}
 
 Return the names of all the columns in the query result.
 """
 function column_names(jl_result::Result)
-    [column_name(jl_result, i) for i in 1:num_columns(jl_result)]
+    return [column_name(jl_result, i) for i in 1:num_columns(jl_result)]
 end
 
 """
-    column_number(jl_result::Result, column_name::AbstractString) -> Int
+    column_number(jl_result::Result, column_name::Union{AbstractString, Symbol}) -> Int
 
 Return the index (1-based) of the column named `column_name`.
 """
-function column_number(jl_result::Result, column_name::AbstractString)::Int
+function column_number(jl_result::Result, column_name::Union{AbstractString, Symbol})::Int
     # todo: check cleared?
-    libpq_c.PQfnumber(jl_result.result, String(column_name)) + 1
+    return libpq_c.PQfnumber(jl_result.result, String(column_name)) + 1
 end
 
+"""
+    column_number(jl_result::Result, column_idx::Integer) -> Int
+
+Return the index of the column if it is valid, or error.
+"""
+function column_number(jl_result::Result, colnum::Integer)::Int
```

Comment:
Would `colidx` or `col_idx` be a better name? I find this easy to read `colnum` as `column`

Reply:
Good point, I'll change it.